cluster/authn-webhook.go

```diff
@@ -0,0 +1,62 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/rancher/rke/docker"
+	"github.com/rancher/rke/hosts"
+	"github.com/rancher/rke/log"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	AuthnWebhookConfigDeployer    = "authn-webhook-config-deployer"
```

Comment:
i think we should add that to the pki package, the problem is that we will end up with a lot of containers to deploy artifacts, so i think there are two choices:
- use one container to deploy all artifacts:
   - cloud config file
   - auth token file

or 
- add the auth token config file to the x509 cert and config list

Reply:
Have created one container deployer for cloud & webhook config files.